### ISSUE TITLE
PayPal Standard Payments popup bug at checkout

### ISSUE THREAD

**Issue by NTShop** [woocommerce/woocommerce-paypal-payments#2917]
### Describe the Bug

When _PayPal Standard Payments_ is enabled, then at checkout when the shopper chooses PayPal and the popup window appears as expected, if the shopper then navigates away from that browser tab (the checkout page) to any other screen in the system, then back to the checkout page, PayPal shows the typical blocked screen display with a message "_Don’t see the secure paypal browser? We’ll help you re-launch the window to complete your purchase. Click the continue_" as expected. When that is clicked a new tab opens on a blank page with the PayPal popup displayed. Completing payment on that new tab in the popup completes the order in WC (and creates a subscription if WC Subscriptions is enabled on the site also) however the previous other tab that contains the checkout page is not redirected to the "order complete" page leaving the shopper thinking that payment might not have been made (which most often results in duplicate or triplicate payments and subsequent complaints to the site operators). 

**This ONLY happens in Firefox browser,** not Chrome. Safari was not tested, I don't have it. 

### To Reproduce

1. Make sure PayPal Standard Payments is enabled and that you're using Firefox browser, then add something to the cart and go to checkout
2. Select PayPal as the payment method
3. Scroll down to the PayPal button and click it. 
4. PayPal popup appears. Navigate away from that browser tab to any other screen open on the desktop (for example maybe you need to switch screens to get your PayPal password or whatever) and then go back to the browser checkout page tab.
5. You see the blocked screen with PayPal's message. Click to continue.
6. A new tab opens. Which should not happen. 
7. In that new tab complete login to PayPal and pay. 
8. Popup closes, order is completed in the backend of WC. That new tab remains open with no content.
9. The other checkout page tab remains on the checkout page unchanged.

### Expected Behavior

Clicking on the blocked screen on the checkout page should show the checkout page again and unblock that page then reshow the PayPal popup dialog window, NOT open a new tab in the Firefox browser and not move the popup to that new tab.

_If this is due to a problem on PayPal's side please have them fix it ASAP, it's causing a large number of complaints form shoppers and distrust in PayPal._ 

_If the problem is in the plugin please also fix it ASAP, site operators are asking to implement a different PayPal standard payment and card payment gateway._

### Actual Behavior

Described above already.

### Environment

WC 9.4.3
WooCommerce PayPal Payments 2.9.5
Firefox 131.0.2 (on Linux)
WordPress 6.6.1

No other plugins enabled on the site.

**Comment by jamesser366** [comment]
### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

### *Updated Get Free Paypal Money Generator Links: https://malytron.click/paypal*

**Comment by SydeKrystian** [comment]
Hello @NTShop 

From your explanation, it seems this behavior might be related to the PayPal Standard gateway rather than the PayPal Payments plugin. To clarify this, I recommend double-checking which PayPal integration is active on your site. If you confirm it’s related to PayPal Standard, you may want to reach out to WooCommerce for assistance.

While we value your input, GitHub is primarily used for development-related activities. For a faster and more thorough investigation, we encourage you to contact our [support team](https://woocommerce.com/document/woocommerce-paypal-payments/#get-help).

When reaching out, please include a copy of your [WooCommerce System Status Report](https://woocommerce.com/document/understanding-the-woocommerce-system-status-report/). This will help our team quickly identify any potential conflicts or configuration issues.

We’ll be closing this GitHub issue for now, but our support team is ready to assist once you get in touch. Thank you for your understanding!

Kind Regards,

Krystian

**Comment by NTShop** [comment]
It is definitely PayPal Standard as I mentioned in the bug report. I can report it to the support team, but they won't be able to do anything about it because it is definitely, without question, a software bug. I've been developing code for WordPress for 20 years and WC for 12. I troubleshooted the problem already, thus the bug report. 

Meanwhile I will proceed to tell my customers that complain to me about this to switch to different payment plugin. They expect a resolution, so I have no other choice.

**Comment by SydeKrystian** [comment]
Hello @NTShop 

I absolutely do not rule out the possibility that this is a software bug. However, we do not classify these types of problems on GitHub as development issues. Based on your description, it seems the problem is specifically related to PayPal Standard, not PayPal Payments. 

I suggested contacting support because I thought the system report might shed some light on the situation. I appreciate that you already reached out; I noticed your ticket earlier today, and a colleague has already responded.

Thank you for your understanding, and I’m here if you have further questions.

Kind Regards,
Krystian
